server.js

```diff
@@ -9,7 +9,21 @@ const app = express();
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json({ limit: '4000mb' }));
 
-app.use(express.static('pack'));
+if(process.env.HOT_RELOAD) {
```

Comment:
the "if" should have a space after it.

Reply:
If yall want to get crazy, you could try adding some automated linting on GitHub.